build.sh

```diff
@@ -154,6 +154,8 @@ else
 	NPROC=$(nproc)
 fi
 
+NPROC=$(($NPROC+1))
```

Comment:
Could we set this to nproc*1.5 ?

Reply:
As the people in the issue reported in the GitHub issue at some point the operations block each other e.g. for 4 cores 6 operations are slower etc. I think we are good with this formula for now (our main goal is to build on GitHub actions anyways), which does have a limit of 2 cores.

Reply:
Ok!